<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HaSwitchWidget</class>
 <widget class="QWidget" name="HaSwitchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>主备状态</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>当前主备状态：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ha_info">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="changeMaster">
          <property name="text">
           <string>切换为主机</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="changeSlave">
          <property name="text">
           <string>切换为备机</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
